BasicPlaybackSampleApp/app/src/main/java/com/ooyala/sample/players/BasicPlaybackVideoPlayerActivity.java

```diff
@@ -95,6 +102,48 @@ public void update(Observable arg0, Object arg1) {
       return;
     }
 
+  // Hook to write Notifications to a temporary file on the device/emulator
+    // Keeps track of incoming notifications and makes sure count is right
+    count++ ;
```

Comment:
Notification received needs to stay as part of the Update().  Also, can you name the file to be more specific? like BasicPlaybackSampleApp.log.

Is there any way to only enable this when you're testing?  I'm just concerned that this will add files to the customer's devices.  This is not a blocking issue, just curious if you can think of something